### PR TITLE
linux/glibc tagged ptr supports for arm64 control addition.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/unix/linux_like/linux/gnu/b64/aarch64/mod.rs [hunk 1]
```diff
@@ -573,6 +573,12 @@ pub const HWCAP_PACG: ::c_ulong = 1 << 31;
 //pub const HWCAP2_SVESM4: ::c_ulong = 1 << 6;
 //pub const HWCAP2_FLAGM2: ::c_ulong = 1 << 7;
 //pub const HWCAP2_FRINT: ::c_ulong = 1 << 8;
+pub const HWCAP2_MTE: ::c_ulong = 1 << 18;
+
+// linux/prctl.h
+pub const PR_GET_TAGGED_ADDR_CTRL: ::c_int = 56;
+pub const PR_GET_TAGGED_ADDR_CTRL: ::c_int = 55;
```
Comment (JohnTitor) [inline src/unix/linux_like/linux/gnu/b64/aarch64/mod.rs:580]:
```suggestion
pub const PR_SET_TAGGED_ADDR_CTRL: ::c_int = 55;
pub const PR_GET_TAGGED_ADDR_CTRL: ::c_int = 56;
```

Comment (devnexen) [reply src/unix/linux_like/linux/gnu/b64/aarch64/mod.rs:580]: Yes too quick copy paste :-)

**Comment by devnexen** [comment]
Above all waiting if the aux vec flag will pass during CI then would make sense to uncomment the rest.

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
```
 error[E0432]: unresolved imports `libc::PR_GET_TAGGED_ADDR_CTRL`, `libc::PR_SET_TAGGED_ADDR_CTRL`, `libc::PR_TAGGED_ADDR_ENABLE`
    --> /checkout/target/aarch64-unknown-linux-musl/debug/build/libc-test-d622df2af2fc66ce/out/semver.rs:4059:5
     |
4059 |     PR_GET_TAGGED_ADDR_CTRL,
     |     ^^^^^^^^^^^^^^^^^^^^^^^ no `PR_GET_TAGGED_ADDR_CTRL` in the root
4060 |     PR_SET_TAGGED_ADDR_CTRL,
     |     ^^^^^^^^^^^^^^^^^^^^^^^ no `PR_SET_TAGGED_ADDR_CTRL` in the root
4061 |     PR_TAGGED_ADDR_ENABLE,
     |     ^^^^^^^^^^^^^^^^^^^^^ no `PR_TAGGED_ADDR_ENABLE` in the root
```

The semver check fails on aarch64-unknown-linux-musl.

**Pull request merged**